acceptance-tests/src/test/java/tech/pegasys/web3signer/dsl/utils/HealthCheckResultUtil.java

```diff
@@ -0,0 +1,82 @@
+/*
+ * Copyright 2023 ConsenSys AG.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+package tech.pegasys.web3signer.dsl.utils;
+
+import java.util.List;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.google.common.base.Splitter;
+
+public class HealthCheckResultUtil {
+
+  private static final ObjectMapper OBJECT_MAPPER = new ObjectMapper();
+
+  /*
+        Healthcheck Json looks like:
+        {
+          "status": "UP",
+          "checks": [
+              {
+                  "id": "default-check",
+                  "status": "UP"
+              },
+              {
+                  "id": "keys-check",
+                  "status": "UP",
+                  "checks": [
+                      {
+                          "id": "wallet-bulk-loading",
+                          "status": "UP",
+                          "data": {
+                              "keys-loaded": 4,
+                              "error-count": 0
+                          }
+                      },
+                      {
+                          "id": "config-files-loading",
+                          "status": "UP",
+                          "data": {
+                              "keys-loaded": 4,
+                              "error-count": 0
+                          }
+                      }
+                  ]
+              }
+          ],
+          "outcome": "UP"
+          }
+  */
+  public static int getHealthCheckKeysCheckData(
```

Comment:
It's not clear what "data" is here

Reply:
I've refactored this class to introduce methods that is more readable/useable in their context in other acceptance tests as well.